listenerutil/parse.go

```diff
@@ -110,8 +110,13 @@ func (l *ListenerConfig) GoString() string {
 	return fmt.Sprintf("*%#v", *l)
 }
 
-func ParseListeners(list *ast.ObjectList) ([]*ListenerConfig, error) {
+func ParseListeners(list *ast.ObjectList, opt ...Option) ([]*ListenerConfig, error) {
```

Comment:
Not your fault but can we add a comment to this function? Make sure to document the supported options.